app/scripts/controllers/swaps.js

```diff
@@ -113,7 +113,7 @@ export default class SwapsController {
     fetchTradesInfo = defaultFetchTradesInfo,
     getCurrentChainId,
     getEIP1559GasFeeEstimates,
-    onNetworkDidChange,
```

Comment:
What do you think about renaming this to `onNetworkIdChange`? We have usually reserved `onNetworkStateChange` for the whole state, but that's not how it would get used here.

Reply:
that makes sense

Reply:
This does depend on more than just the ID though. It uses the status as well.

Reply:
yeah, but it is explicitly listening for the ID change

Reply:
but as per Elliot's other comment below, maybe we should just listen for all state changes

Reply:
I'm not sure what you mean by that.

A change to either property can trigger the effect. If it were `onNetworkIdChange`, I'd expect it to only be triggered when the ID changes. That would result in this potentially being stuck on the wrong network if the status was anything other than "available" when the id changed.